systemdspawner/systemd.py

```diff
@@ -143,6 +144,20 @@ async def start_transient_service(
         )
         run_cmd.append(f"--property=EnvironmentFile={environment_file}")
 
+    # make sure cmd[0] is absolute, which is a requirement of systemd-run
+    # if it's relative, resolve it with Spawner's $PATH,
+    # as any other Spawner would behave
+    if not os.path.isabs(cmd[0]):
+        path = os.getenv("PATH", os.defpath)
```

Comment:
```suggestion
```
Leftover from testing?

Reply:
Yes, but it's the next line that should be removed, not this one.